model/config.go

```diff
@@ -293,6 +293,7 @@ type ServiceSettings struct {
 	EnableEmailInvitations                            *bool
 	ExperimentalLdapGroupSync                         *bool
 	DisableBotsWhenOwnerIsDeactivated                 *bool `restricted:"true"`
+	CreateBotAccounts                                 *bool
```

Comment:
@crspeller, should this be something that's restricted for Mattermost Cloud?

Reply:
Ideally no. Currently only system admins can create bots even when this setting is enabled. When it gets added to the permissions UI it should be able to be controlled from there.